### PR TITLE
Handwritten Digit Recognition

### LINKED ISSUE #14: Handwritten Digit Recognition
Implement a digit recognition system that can recognize handwritten digits (0-9). You can use the MNIST dataset for this project.

### PR DESCRIPTION
**Pull request by ishan-nahid**
# Description

In this project, we will walk through the process of building a handwritten digit recognition model using the
MNIST dataset and TensorFlow. We will load and preprocess the data, create a neural network, train the
model, and evaluate its performance.

## Issue

Fixes #14 

## Type of change
- [ ] Bug fix
- [x] New Project

# Checklist:

- [x] I have included a REAME file for my Project (not for Bug fix)
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas 
- [x] I have followed the [Contributing Guidelines](https://github.com/Prodigy-InfoTech/.github/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ishan-nahid** [comment]
@prodigyinfotech check, feedback and marge with "hacktoberfest-accepted" label.

**Comment by prodigyinfotech** [comment]
do not use dash in foldername

**Comment by ishan-nahid** [comment]
should i submit another PR just renaming the folder name? 
or you will marge it then rename it??

**Comment by prodigyinfotech** [comment]
rename the folder in your repository and the commit will add up in the PR automatically

**Comment by ishan-nahid** [comment]
check now. 
@prodigyinfotech
